### PR TITLE
PUBDEV-7562 consolidate compare frame to one method

### PR DESCRIPTION
**Pull request by wendycwong**
This PR completes the task in JIRA: https://0xdata.atlassian.net/browse/PUBDEV-7562

In Python pyunit_utils.py, the compare_frames will ask for frame elements one entry at a time from Java backend, this is not efficient.  I have changed the following methods to grab whole chunks of data from h2o backend instead of one entry at a time

-compare_numeric_frames : deleted
-compareOneNumericColumn: deleted
-compareOneStringColumn: deleted
-compare_frames: call compare_one_column_local method now

### CODE REVIEW

File: h2o-py/tests/pyunit_utils/utilsPY.py [hunk 1]
```diff
@@ -2994,83 +2995,14 @@ def compare_frames(frame1, frame2, numElements, tol_time=0, tol_numeric=0, stric
             custom_comparators[c1_key](frame1, frame2, col_ind, rows1, numElements)
         elif (str(c1_type) == 'string') or (str(c1_type) == 'enum'):
             # compare string
-            compareOneStringColumn(frame1, frame2, col_ind, rows1, numElements)
+            compare_frames_local_onecolumn_NA_string(frame1[col_ind], frame2[col_ind], prob=probVal)
         else:
             if str(c2_type) == 'time':  # compare time columns
-                compareOneNumericColumn(frame1, frame2, col_ind, rows1, tol_time, numElements)
+                compare_frames_local_onecolumn_NA(frame1[col_ind], frame2[col_ind], prob=probVal,tol=tol_time)
```
Comment (honzasterba) [inline h2o-py/tests/pyunit_utils/utilsPY.py:3001]: fix whitespace (space after ,)

File: h2o-py/tests/pyunit_utils/utilsPY.py [hunk 2]
```diff
@@ -2994,83 +2995,14 @@ def compare_frames(frame1, frame2, numElements, tol_time=0, tol_numeric=0, stric
...
             else:
-                compareOneNumericColumn(frame1, frame2, col_ind, rows1, tol_numeric, numElements)
+                compare_frames_local_onecolumn_NA(frame1[col_ind], frame2[col_ind], prob=probVal,tol=tol_numeric)
```
Comment (honzasterba) [inline h2o-py/tests/pyunit_utils/utilsPY.py:3003]: fix whitespace (space after ,)

File: h2o-py/tests/pyunit_utils/utilsPY.py [hunk 3]
```diff
@@ -2968,6 +2968,7 @@ def compare_frames(frame1, frame2, numElements, tol_time=0, tol_numeric=0, stric
```
Comment (honzasterba) [inline h2o-py/tests/pyunit_utils/utilsPY.py:2968]: Ideally I would merge the compare_frames and compare_frames_local into one function, but I can see that would be big task

File: h2o-py/tests/pyunit_utils/utilsPY.py (hunk 1 above)
Comment (wendycwong) [reply h2o-py/tests/pyunit_utils/utilsPY.py:3001]: Done

File: h2o-py/tests/pyunit_utils/utilsPY.py (hunk 2 above)
Comment (wendycwong) [reply h2o-py/tests/pyunit_utils/utilsPY.py]: Done

File: h2o-py/tests/pyunit_utils/utilsPY.py (hunk 3 above)
Comment (wendycwong) [reply h2o-py/tests/pyunit_utils/utilsPY.py:2968]: Yeah and I am lazy.

Comment (honzasterba) [reply h2o-py/tests/pyunit_utils/utilsPY.py:2968]: what about making compare_frames_local call compare_frames? that would be good middle of the road solution

**Pull request merged**